src/views/pages/models.jsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import ListenerComponent from '../../lib/listener-component.jsx';
+import ModelStore from '../../store/model-store.js';
+import CreateModelForm from '../components/create-model-form.jsx';
+import { getAllModels } from '../../lib/api-client'
+export default class Models extends ListenerComponent {
+
+    constructor() {
```

Comment:
Shouldn't need to override the constructor if you're only going to call `super`.